package.json

```diff
@@ -27,7 +27,7 @@
     "browserify-rsa": "^4.0.1",
     "create-hash": "^1.2.0",
     "create-hmac": "^1.1.7",
-    "elliptic": "^6.5.3",
```

Comment:
there's simply no need for this update; we depend on `elliptic` with a semver range, so you can just update your lockfiles.